kong-ngx-build

```diff
@@ -413,7 +413,7 @@ parse_version() {
 
   local subj=$1
   local -n ver=$2
-  local re='^(-rc[0-9]+$)?[^0-9]*([0-9a-zA-Z]*)(.*?)$'
+  local re='^(-rc[0-9]+$)?[^0-9]*([0-9a-zA-Z]*)(.*)$'
```

Comment:
@eskerda Is this change working on your side?

Reply:
Yup, seems to fix it